cuegui/cuegui/HostMonitor.py

```diff
@@ -51,18 +52,21 @@ def __init__(self, parent):
 
         # This hlayout would contain any filter/control buttons
         hlayout = QtWidgets.QHBoxLayout()
-        self.__filterByHostNameSetup(hlayout)      # Menu to filter by host name
-        self.__filterAllocationSetup(hlayout)    # Menu to filter by allocation
-        self.__filterHardwareStateSetup(hlayout) # Menu to filter by hardware state
+        self.__filterByHostNameSetup(hlayout)     # Menu to filter by host name
+        self.__filterAllocationSetup(hlayout)     # Menu to filter by allocation
+        self.__filterHardwareStateSetup(hlayout)     # Menu to filter by hardware state
         hlayout.addStretch()
-        self.__refreshToggleCheckBoxSetup(hlayout) # Checkbox to disable auto refresh
-        self.__refreshButtonSetup(hlayout)         # Button to refresh
-        self.__clearButtonSetup(hlayout)           # Button to clear all filters
+        self.__refreshToggleCheckBoxSetup(hlayout)     # Checkbox to enable/disable auto refresh
+        self.__refreshButtonSetup(hlayout)     # Button to refresh
+        self.__clearButtonSetup(hlayout)     # Button to clear all filters
 
         self.layout().addLayout(hlayout)
         self.layout().addWidget(self.hostMonitorTree)
 
-        self.__viewHostsSetup()                  # For view_hosts signal
+        self.__viewHostsSetup()     # For view_hosts signal
+        
+        if bool(QtGui.qApp.settings.value("AutoRefreshMonitorHost", 1)):     # For refresh on launch
```

Comment:
I believe this line needs to convert to `int` as well.

Reply:
Yes, you are right. My mistake.

Reply:
correction made.

Reply:
I believe `bool(int(x)))` call is redundant, is there any benefit over simply allowing Python to handle the truthiness check?
for example:
`if int(QtGui.qApp.settings.value("AutoRefreshMonitorHost", 1)): ...` 
has the same effect as 
`if bool(int(QtGui.qApp.settings.value("AutoRefreshMonitorHost", 1))): ...`

Reply:
@gregdenton sir, you are right. I have used bool() to make the mechanism more clear to understand. It doesn't have any significance as far as working of code is concerned. It could be removed if it is looking bit too heavy-handed.
IMO it is good to store and use bool value for conditional statements.
For example bool value stored below:
`self.enableRefresh = bool(int(QtGui.qApp.settings.value("AutoRefreshMonitorHost", 1)))`

Reply:
Yeah, I'm a bit torn on this one. The `bool()` is technically not necessary but it does make it more explicit, which is especially useful in a class-wide variable like in @srbhss's example.

I think I would lean towards keeping the bool cast.